File: Ejercicios/Java/Ejercicio1.java
import java.util.ArrayList;
public class Ejercicio1 {
    public static void main(String[] args) {
        ArrayList<String> estudiantes = new ArrayList<>();
        estudiantes.add("Victor");
        estudiantes.add("Maria");
        estudiantes.add("Luis");
        estudiantes.add("Ana");
        estudiantes.add("Pedro");
        for(int i = 0; i < estudiantes.size(); i++) {
            System.out.println("Estudiante " + (i+1) + ": " + estudiantes.get(i));
        }
    }
}

File: Ejercicios/C++/Ejercicio1.cpp
#include <iostream>
#include <vector>
using namespace std;
int main() {
    vector<string> estudiantes;
    estudiantes.push_back("Victor");
    estudiantes.push_back("Maria");
    estudiantes.push_back("Luis");
    estudiantes.push_back("Ana");
    estudiantes.push_back("Pedro");
    for(int i = 0; i < estudiantes.size(); i++) {
        cout << "Estudiante " << i+1 << ": " << estudiantes[i] << "\n";
    }
    return 0;
}
